pkg/api/server/managementstored/setup.go

```diff
@@ -347,3 +350,12 @@ func Pipeline(schemas *types.Schemas, management *config.ScaledContext) {
 	schema.Formatter = pipeline.SourceCodeRepositoryFormatter
 	schema.LinkHandler = sourceCodeRepositoryHandler.LinkHandler
 }
+
+func Project(schemas *types.Schemas, management *config.ScaledContext) {
+	schema := schemas.Schema(&managementschema.Version, client.ProjectType)
+	schema.Formatter = projectaction.Formatter
+	handler := &projectaction.Handler{
+		ProjectClient: management.Management.Projects(""),
```

Comment:
We ~have standardized~ are trying to standardize on the plural of the noun to represent a client. Also, a struct field should only be capitalize if it is going to be used outside of its package. So, I think this should be `projects` instead of `ProjectClient`